app/index.js

```diff
@@ -14,17 +13,16 @@ const app = express()
 app.use(express.static(distPath))
 
 // setup app configurations
-app.disable('x-powered-by')
-
 const hostName = require('os').hostname()
 const compression = require('compression')()
+app.disable('x-powered-by')
```

Comment:
what is this doing?

Reply:
It is a small security measure / best practice to hide that this is an express powered server. An attacker could use this as hint and know in advance which vulnerabilities he could try to exploit something :)